ledger/storage/db/badger_test.go

```diff
@@ -1,17 +1,97 @@
+/*
+ *    Copyright 2019 Insolar
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
 package db
 
 import (
+	"io/ioutil"
 	"testing"
+
+	"github.com/dgraph-io/badger"
+	"github.com/google/gofuzz"
+	"github.com/insolar/insolar/configuration"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
+type testBadgerKey struct {
+	id    []byte
+	scope Scope
+}
+
+func (k testBadgerKey) Scope() Scope {
+	return k.scope
+}
+
+func (k testBadgerKey) ID() []byte {
+	return k.id
+}
+
 func TestBadgerDB_Get(t *testing.T) {
 	t.Parallel()
 
-	// tmpdir, err := ioutil.TempDir("", "bdb-test-")
-	// assert.NoError(t, err)
-	//
-	// bdb, err := badger.Open(badger.Options{
-	// 	Dir:      tmpdir,
-	// 	ValueDir: tmpdir,
-	// })
+	tmpdir, err := ioutil.TempDir("", "bdb-test-")
```

Comment:
This tmp-dir should be removed.
Here citation from doc for  `ioutil.TempDir`
```
It is the caller's responsibility
to remove the directory when no longer needed.
```
(edit here and below)